statefun-python-sdk/examples/k8s/Dockerfile.python-worker

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.7-alpine
+
+RUN mkdir -p /app
+WORKDIR /app
+
+# The following two lines would be removed once statefun would be published to PyPI
```

Comment:
Do you intend to let this example work with a locally built Python SDK?
If so, I think the example should follow the Java examples, where by default they work without pulling packages from Maven central.

Maybe adding an `LOCAL_WHL_PATH` option which by default is set to the path of the locally built Python SDK distribution?